### DISCUSSION TITLE
YM2608-LLE implementation and emulation core selector rework

### DISCUSSION THREAD

**Discussion by freq-mod** [LTVA1/furnace#90] [Ideas]
YM2608-LLE is getting mature, so it can get implemented for rendering. However, some problems are:
YM2608-LLE emulates YM2608, 2610 and 2612, and not YM2203 and YM3438. So, either we ignore that and use 2608 LLE core for 2203 too (as YM2608 boots to 2203 mode by default anyway) or separate YM2203 entry. Also, YM2612 core selector would have to be separate for YM2612 proper (which is covered by YM2608-LLE) and YM3438/F276 (which are covered by YMF276-LLE and Nuked-OPN2)

**Comment by LTVA1** [comment]
Yes, we would need the core selector rework.
I don't like the idea of using YM2608 emulator for YM2203 since there may be some miniscule differences and with LLE cores we aim at sample-accurate emulation, so better safe than sorry.
